autoinstall-schema.json

```diff
@@ -139,6 +139,23 @@
             ],
             "additionalProperties": false
         },
+        "zdevs": {
```

Comment:
Every commit that modifies the schema should be matched with an autoinstall-reference update, please.